Components/Sensors/MTK3339.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System;
+using System.Diagnostics;
+using System.Text;
+using System.Threading;
+using Scarlet;
+using Scarlet.Components;
+using Scarlet.IO;
+using Scarlet.IO.BeagleBone;
+
+namespace UseRobot
+{
+    public class MTK3339 : ISensor
+    {
+        string UPDATE_200_MSEC = "$PMTK220,200*2C\r\n";
+        string MEAS_200_MSEC = "$PMTK300,200,0,0,0,0*2F\r\n";
+        string GPRMC_GPGGA = "$PMTK314,0,1,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0*28\r\n";
+        string QUERY_GPGSA = "$PSRF103,02,01,00,01*27\r\n";
+
+        public bool Fix
+        {
+            get
+            {
+                write_string(QUERY_GPGSA);
+                string[] result = read();
+                if (result.Length < 3)
```

Comment:
Even if it's one line, please surround with brackets to avoid potential issues in the future.